Application/src/visuals/DiagramClass.java

```diff
@@ -9,22 +9,29 @@
  * @author Pontus Laestadius
  */
 public class DiagramClass implements Renderable {
-    private Coordinates coordinates;
-    private int size;
-    private String name;
+    private Coordinates coordinates; // The coordinates of the object.
+    private int size; // Size of the Object.
+    private String name; // The class name.
+
+    // The images retrieved.
+    private static Image castle1 = new Image("resources/castle_default.png");
+    private static Image castle2 = new Image("resources/castle_default2.png");
+
+    private static Image platform = new Image("resources/platform_default.png");
+    private static Image connector = new Image("resources/connector_default.png");
+    private static Image pillar = new Image("resources/pillar_default.png");
 
     // Animations
-    private int aniindex = 0; // Animation index used for writing the state of the animating sequence.
-    private Coordinates[] anicontent; // The sequence that the object follows when animated.
 
-    private static Image castle = new Image("resources/castle.png");
-    private static Image platform = new Image("resources/platform.png");
-    private static Image pillar = new Image("resources/pillar.png");
+    // The states of the castle.
+    private Image[] states = {castle1, castle2};
```

Comment:
Wouldnt castleStates be better since we are going to add more animations soon

Reply:
Not to the Class though. These are all the states of the castles.

Reply:
I can rename it, but It wont get anymore states.